corehq/apps/export/models/new.py

```diff
@@ -651,6 +651,14 @@ def get_column(self, item_path, item_doc_type, column_transform):
 
         return None, None
 
+    def get_column_by_path_str(self, path, doc_type):
+        path_nodes = [PathNode(name=node) for node in path.split('.')]
+        if not path_nodes:
```

Comment:
in what scenario would path_nodes be falsy here?

Reply:
Yeah, I suppose it won't since the result will just simply return whatever is passed in.